structuralcodes/codes/mc2010/_concrete_creep_and_shrinkage.py

```diff
@@ -0,0 +1,818 @@
+
+    returns:
+        float: Exponent gamma_t0.
+    """
+    return 1 / (2.3 + 3.5 / np.sqrt(t0_adj))
+
+
+def _calc_beta_dc_t(
+    time: np.ndarray, t0: float, beta_h: float, gamma_t0: float
+) -> np.ndarray:
+    """Calculate multiplication factor that accounts for the different
+        considered values of time. Used to calculate the drying creep
+        coefficient.
+
+    Defined in fib Model Code 2010 (2013), Eq. 5.1-71a.
+
+    args:
+        time (numpy.ndarray): The different times in days at which the
+            drying creep coefficient is determined.
+        t0 (float): The age of the concrete concrete
+            when the loading is applied in days.
+        beta_h (float): Multiplication factor that accounts for the effect of
+            the notional size, as calculated by Eq. 5.1-71c
+        gamma_t0 (float): Exponent that accounts for the effect of the
+            (temperature corrected) age of the concrete when loaded,
+            as calculated by Eq. 5.1-71b.
+
+    returns:
+        numpy.ndarray: Multiplcation factor beta_dc_t for the
+            considered values of time.
+    """
+    return ((time - t0) / (beta_h + (time - t0))) ** gamma_t0
+
+
+def calc_drying_creep_coefficient(
+    beta_dc_fcm: float,
+    beta_dc_RH: float,
+    beta_dc_t0: float,
+    beta_dc_t: np.ndarray,
+) -> np.ndarray:
+    """Calculate drying creep coefficient.
+
+    Defined in fib Model Code 2010 (2013), Eq. 5.1-67.
+
+    args:
+        beta_dc_fcm (float): Multiplication factor that accounts for
+            the effect of the strength of the concrete, as calculated
+            by Eq. 5.1-68.
+        beta_dc_RH (float): Multiplication factor that accounts for the
+            effect of the relative humidity of the environment, as
+            calculated by Eq. 5.1-69.
+        beta_dc_t0 (float): multiplication factor that accounts for
+            the effect of the (temperature corrected) age of the
+            concrete when loading is applied, as calculated by Eq.
+            5.1-70.
+        beta_dc_t (numpy.ndarray): multiplication factor that accounts
+            for the different considered values of time, as calculated
+            by Eq. 5.1-71a.
+
+    returns:
+        numpy.ndarray: Drying creep coeffcient.
+    """
+    return beta_dc_fcm * beta_dc_RH * beta_dc_t0 * beta_dc_t
+
+
+def _calc_k_sigma(sigma: float, fcm: float) -> float:
+    """Calculate the ratio between the applied stress and the mean
+        concrete compressive strength.
+
+    Defined in fib Model Code 2010 (2013), Eq. 5.1-74.
+
+    args:
+        sigma (float): The compressive stress applied to the concrete
+            at ts in MPa.
+        fcm (float): The mean compressive strength of the concrete in
+            MPa.
+
+    returns:
+        float: Absolute value of the ratio between the stress in the
+            concrete and the mean concrete strength.
+    """
+    return abs(sigma / fcm)
+
+
+def calc_creep_coefficient(
```

Comment:
the only inputs parameters should be phi_bc and phi_dc. We already have functions to compute them.
The body of the functions simplifies greatly.
The logic of this current function (as in previous cases) is more related to the concreteMC2010 class.